wiki/Ranking_Criteria/Difficulty_Naming/en.md

```diff
@@ -0,0 +1,107 @@
+
+### Beatmania IIDX (or BMS songs)
+
+- ![Beginner](/wiki/shared/diff/easy-s.png "Beginner") Beginner
+- ![Normal](/wiki/shared/diff/normal-s.png "Normal") Normal
+- ![Hyper](/wiki/shared/diff/hard-s.png "Hyper") Hyper
+- ![Another](/wiki/shared/diff/insane-s.png "Another") Another
+- ![Black Another](/wiki/shared/diff/expert-s.png "Black Another") Black Another
+
+### SOUND VOLTEX (SDVX)
+
+Abbreviations on this particular difficulty naming scheme are acceptable (e.g. NOVICE → NOV).
+
+- ![BASIC](/wiki/shared/diff/easy-s.png "BASIC") BASIC 
+- ![NOVICE](/wiki/shared/diff/normal-s.png "NOVICE") NOVICE
+- ![ADVANCED](/wiki/shared/diff/hard-s.png "ADVANCED") ADVANCED
+- ![EXHAUST](/wiki/shared/diff/insane-s.png "EXHAUST") EXHAUST
+- ![INFINITE](/wiki/shared/diff/expert-s.png "INFINITE") INFINITE
+
+### Pop 'n Music
+
+- ![Easy](/wiki/shared/diff/easy-s.png "Easy") Easy 
+- ![Normal](/wiki/shared/diff/normal-s.png "Normal") Normal
+- ![Hyper](/wiki/shared/diff/hard-s.png "Hyper") Hyper
+- ![Ex](/wiki/shared/diff/insane-s.png "Ex") Ex 
+
+### In The Groove (ITG)
+
+- ![Novice](/wiki/shared/diff/easy-s.png "Novice") Novice
+- ![Easy](/wiki/shared/diff/easy-s.png "Easy") Easy
+- ![Medium](/wiki/shared/diff/normal-s.png "Medium") Medium
+- ![Hard](/wiki/shared/diff/hard-s.png "Hard") Hard
+- ![Expert](/wiki/shared/diff/insane-s.png "Expert") Expert
+
+## Songs from other Games
+
+### Touhou Project *(東方Project)*
+
+Difficulty spread inspired by Touhou Project (東方Project in Japanese), including any individual game from the project.
+
+The difficulty spread would follow the Default difficulty naming scheme, with the exception of the Insane difficulty, which would be substituted by Lunatic.
+
+- ![Lunatic](/wiki/shared/diff/insane-s.png "Lunatic") Lunatic
+
+## Widely Accepted Alternatives 
+
+The following **individual** difficulty names have been accepted and are used more commonly now as alternative to the basic difficulty names:			
+
+### ![osu!](/wiki/shared/Osu.gif "osu!") osu!
+
+- ![Beginner](/wiki/shared/diff/easy-s.png "Beginner") **Beginner**: in a default Easy-Normal-Hard-Insane-Expert spread, it means the lowest level even below Easy.
+- ![Advanced](/wiki/shared/diff/normal-s.png "Advanced") **Advanced**: in a default Easy-Normal-Hard-Insane-Expert spread, it's a difficulty between Normal and Hard.
+- ![Hyper](/wiki/shared/diff/hard-s.png "Hyper") **Hyper**: in a default Easy-Normal-Hard-Insane-Expert spread, a difficulty between Hard and Insane.
+- ![Extreme](/wiki/shared/diff/expert-s.png "Extreme") **Extreme**: can be used to represent a difficulty tier above Extra/Expert.
+
+### ![osu!taiko](/wiki/shared/Taiko.gif "osu!taiko") osu!taiko
+
+- ![Hell Oni](/wiki/shared/diff/expert-t.png "Hell Oni") **Hell Oni**: commonly accepted as a difficulty that is a tier above Inner Oni / Ura Oni.
+
+### ![osu!catch](/wiki/shared/Ctb.gif "osu!catch") osu!catch
```

Comment:
Spoon is generally accepted as a tier below Cup

Reply:
been personally asked not to add this one because "anything below cup is still cup, no dashing required, no unique feature to differentiate from a cup"

Reply:
Spoon diffs were a mistake